qemu/tests/numa_hmat.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from virttest import utils_misc
+from virttest import env_process
+
+from virttest.utils_version import VersionInterval
+
+
+def run(test, params, env):
+    """
+    Acpi hmat test
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    qemu_path = utils_misc.get_qemu_binary(params)
+    qemu_version = env_process._get_qemu_version(qemu_path)
+    host_qemu = re.search(r'[0-9]+\.[0-9]+\.[0-9]+(\-[0-9]+)?',
+                          qemu_version).group(0)
+    if host_qemu in VersionInterval('[,5.2.0)'):
+        params['numa_hmat_caches_size_hmat_cache1'] = '50K'
+        params['numa_hmat_caches_size_hmat_cache2'] = '40K'
+        params['numa_hmat_caches_size_hmat_cache3'] = '80K'
+        params['numa_hmat_caches_size_hmat_cache4'] = '70K'
+        params['numa_hmat_caches_size_hmat_cache5'] = '60K'
+    else:
+        params['numa_hmat_caches_size_hmat_cache1'] = '40K'
+        params['numa_hmat_caches_size_hmat_cache2'] = '50K'
+        params['numa_hmat_caches_size_hmat_cache3'] = '60K'
+        params['numa_hmat_caches_size_hmat_cache4'] = '70K'
+        params['numa_hmat_caches_size_hmat_cache5'] = '80K'
```

Comment:
Test failed.

22:50:19 INFO | [qemu output] qemu-kvm: -numa hmat-cache,node-id=1,level=2,size=40K,associativity=direct,policy=write-back,line=8: Invalid size=40960, the size of level=2 should be larger than the size(51200) of level=1
22:50:19 INFO | [qemu output] (Process terminated with status 1)